modoboa/core/constants.py

```diff
@@ -36,6 +36,7 @@
     ("nl", u"nederlands"),
     ("pt_PT", u"português"),
     ("pt_BR", u"português (BR)"),
+    ("pl_PL", u"polski"),
     ("ru", u"русский"),
```

Comment:
Hello,why am I added Chinese here, but there is no Chinese option in Settings

Reply:
Which code do you use? Are you running the development server?

Reply:
In fact,I don't know the answer,because I just came to this company for few days and my boss asked me to do this.So,maybe I should change the question,when will you have Chinese?

Reply:
When someone will translate Modoboa to Chinese.

Reply:
okay ! thank you .